db/python/layers/sequence.py

```diff
@@ -316,22 +316,44 @@ async def update_sequence_from_sample_and_type(
         return sequence_id
 
     async def update_sequence_from_external_id_and_type(
```

Comment:
Sorry, another nitpick, can you rename this to `upsert_sequence...`

Reply:
Done!